server/src/main/java/io/unitycatalog/server/service/iceberg/FileIOFactory.java

```diff
@@ -42,6 +42,9 @@ public FileIOFactory(CredentialOperations credentialOps) {
 
   // TODO: Cache fileIOs
   public FileIO getFileIO(URI tableLocationUri) {
+    if (tableLocationUri.getScheme() == null) {
```

Comment:
When is it null?

Reply:
when the path is provided without in a non URI format i.e. "/local/path/to/file" as opposed to "file:///local/path/to/file"